Compfiles/Usa2022P3.lean

```diff
@@ -0,0 +1,275 @@
+/-
+Copyright (c) 2024 The Compfiles Contributors. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhiyi Luo
+-/
+
+import Mathlib.Data.Real.Basic
+import Mathlib.Data.NNReal.Basic
+import Mathlib.Algebra.Order.Positive.Field
+import Mathlib.Tactic
+
+import ProblemExtraction
+
+problem_file { tags := [.NumberTheory] }
```

Comment:
I would tag this problem as `Algebra`, not `NumberTheory`.

Reply:
Fixed